src/background.js

```diff
@@ -432,8 +432,7 @@ async function updateBrowserActionIcon (tab) {
 
   const url = tab.url;
   const hasBeenAddedToFacebookContainer = await isAddedToFacebookContainer(url);
-
-
+  const aboutPageURLCheck = url.indexOf("about:");
```

Comment:
```suggestion
  const aboutPageURLCheck = url.startsWith("about:");
```

So we don't get false positives on pages like https://wiki.mozilla.org/Firefox/Projects/about:newtab.